DEVELOPMENT.md

```diff
@@ -0,0 +1,74 @@
+# Setting up newsroom for development
+
+## Repositories
+
+Use `develop` branch of `newsroom-core` repository and `master` branch of `superdesk-app`.
```

Comment:
Typo: `superdesk-app` should be `newsroom-app`

Reply:
We should also be using the `develop` branch from `newsroom-app`, as that is what `nra-develop` instance is built off. If there is an issue with this branch, we should fix it